extensions/amp-ima-video/0.1/test/validator-amp-ima-video-dock-no-extension.html

```diff
@@ -0,0 +1,40 @@
+<!--
```

Comment:
Let's update https://github.com/ampproject/amphtml/blob/master/spec/amp-video-interface.md and remove experimental wording from it as well.

Also update all the supported video's `.md` files with the `dock` attribute similar to autoplay.

Reply:
(separate PR okay)

Reply:
Cool, #20788